### ISSUE TITLE
Convert line to satin: reduce rung count

### ISSUE THREAD

**Issue by gusvisser** [inkstitch/inkstitch#2915]
As i was working with a number of stroke to satin conversions and tried some zigzag to satin i ended up with a lot of unneeded rungs so i was thinking if it would be possible if we could set a number of rungs for our satin created

Gus Visser

**Comment by kaalleen** [comment]
Is this feature request for line to satin, zigzag to satin or both?

**Comment by gusvisser** [comment]
I would prefer specially on line to satin
i have notized that by zigzag to satin that it is contolled by the sawtooth or the square on how the rungs are placed but if it is possible would be nice also

**Comment by kaalleen** [comment]
> I have notized that by zigzag to satin that it is contolled by the sawtooth or the square on how the rungs are placed but if it is possible would be nice also

That's true, the way it is controlled. If you don't want to start out without any rungs, you can even disable the rung creation in the zigzag extension.

**Comment by kaalleen** [comment]
Newly discussed and this can be closed.
